### PR TITLE
[4.0] UX improvements admintemplate

### PR DESCRIPTION
**Pull request by angieradtke**
### Summary of Changes
First step of  visuell cleaning up admin template. Changes (colours, spaces, font-sizes, ...) in scss for a better UX of the template. 
Not finished yet. Lack of hover-colors etc. 
Most changes made in the edit-article-view and at the sidenav.

### Testing Instructions
Just check and report back if you find problem in the listed areas.

### Expected result
Happy users

### CODE REVIEW

**Comment by brianteeman** [comment]
Thanks so much for taking this on.

**Comment by angieradtke** [comment]
@dgrammatiko @Hackwar @wilsonge @HLeithner more conversation here :-)

**Comment by rdeutz** [comment]
Welcome and congratulations for your first PR :-)

**Comment by brianteeman** [comment]
.com_login .sidebar-wrapper .main-brand

change from atum-text-light

**Comment by brianteeman** [comment]
.quick-icons .quickicon a.success

change from color: fff

**Comment by brianteeman** [comment]
image

.header .fa-angle-down ==> increase margin-left

**Comment by angieradtke** [comment]
Stop Brian, it is work in progress. Robert told me I have to make that pull-request now. I'm not finished.
Please take an  overview first and tell me if I go in the right direction

**Comment by brianteeman** [comment]
OK. will do
Looks awesome so far - I am only seeing very minor tweaks

**Comment by brianteeman** [comment]
So the only real things that are not "tweaks" would be the box-shadow on the inputs and the delay on the transitions for the buttons - see below - otherwise keep up the good work

### Now
image

### Better???
image


Animation/hover looks off - maybe the icon and the button should change at the same time

anguie

**Comment by bembelimen** [comment]
Thank you @angieradtke for your work.

**Comment by angieradtke** [comment]
The width of the input fields border is important for accessibillty.  Shall we make them and the boxshadow configurable? Can be  a nice accessibility-feature with less  effort.

**Comment by dgrammatiko** [comment]
> Animation/hover looks off - maybe the icon and the button should change at the same time

Transitions should have the same value, something like:
```css
  transition: opacity 0.2s ease;
```

Another trick I do on the clickable items is:
```css
/* Normal state */
  transform: scale(1);

/* Clicked state */
  transform: scale(0.95);
```

**Comment by brianteeman** [comment]
> The width of the input fields border is important for accessibillty. Shall we make them and the boxshadow configurable? Can be a nice accessibility-feature with less effort.

I agree when you have focus on the input that it should have some clear highlighting but I dont see the need for it on everything all the time. Plus currently the shadow changes from blue to grey on focus so that is failing a11y ;)

**Comment by angieradtke** [comment]
Do we need the sidebar on the login-screen?

**Comment by brianteeman** [comment]
Personally not and the links are silly. But others were very insistent on that

**Comment by angieradtke** [comment]
> 
> 
> Personally not and the links are silly. But others were very insistent on that

I think  we shouldn't give  more infos as needed at that place, because it could be seen by not authorized persons

**Comment by angieradtke** [comment]
cancel
Where can I fix  the missing cancel  button at the new -article- view?

**Comment by dgrammatiko** [comment]
> Do we need the sidebar on the login-screen?

Her's an idea:
- since for whatever reasons some people need those links add them in a detail element with a title Help or something similar after the login button. Clean approach

**Comment by brianteeman** [comment]
> Where can I fix the missing cancel button at the new -article- view?

You put it in the "existing records" part of the if/else checking for new items

**Comment by dgrammatiko** [comment]
Where can I fix the missing cancel button at the new -article- view?

It's the if/else that needs patching 

https://github.com/joomla/joomla-cms/blob/92727dd90a8604bfcd12b5fba81d54bc168799e2/administrator/components/com_content/src/View/Article/HtmlView.php#L222

File: administrator/components/com_content/View/Article/HtmlView.php [hunk 1]
```diff
@@ -156,6 +156,7 @@ function (Toolbar $childBar)
 					$childBar->save2new('article.save2new');
 				}
 			);
+			$toolbar->cancel('article.cancel', 'JTOOLBAR_CLOSE');
```
Comment (richard67) [inline administrator/components/com_content/View/Article/HtmlView.php]:
```suggestion

			$toolbar->cancel('article.cancel', 'JTOOLBAR_CLOSE');
```

**Comment by richard67** [comment]
@angieradtke In future when we have to make a new admin template for J5, can we order that directly from you? That would save us a lot of time ;-)

**Comment by richard67** [comment]
@angieradtke Since PR #27687 has been merged, some files have moved to other place (subdirectory "src"), e.g. file `administrator/components/com_content/View/Article/HtmlView.php` has bee moved to `administrator/components/com_content/src/View/Article/HtmlView.php`. So this PR here can't be easily applied on the current 4.0-dev branch.

Do you think you can fix that? It might not be easy due to merge conflicts. In this case let us know here and I am sure everybody who is able to and reads it and has a bit time will do all to suport you.

**Comment by angieradtke** [comment]
> Do you think you can fix that? It might not be easy due to merge conflicts. In this case let us know here and I am sure everybody who is able to and reads it and has a bit time will do all to suport you.

Tomorrow Robert will help me to make all these fixes, so that I have more time to care about the things I know about  .-) For every help I'm thankful

**Comment by richard67** [comment]
We have to be thankful to you.

**Comment by angieradtke** [comment]
filter

Where can I move  the filter options in front of the search?

**Comment by dgrammatiko** [comment]
> Where can I move the filter options in front of the search?

Either 
https://github.com/joomla/joomla-cms/blob/4.0-dev/layouts/joomla/searchtools/default.php
or any of the files in the subdirectory (don't remember from head)

**Comment by angieradtke** [comment]
What do you think?

switcher

and where to change  if you like  System-Template or Autum?

**Comment by astridx** [comment]
@angieradtke Do you mean adding `&tmpl=component` to the url or like here https://github.com/joomla/joomla-cms/blob/24265b1d085494beb1dafb3e6be32e3d1d50a16b/plugins/editors/tinymce/tinymce.php#L325?

**Comment by angieradtke** [comment]
@astridx 
no we can change it here:  \build\media_source\system\scss\fields\switcher.scss 
or override this in the template

**Comment by angieradtke** [comment]
@astridx sorry I'm talking about the switcher

**Comment by rdeutz** [comment]
If we agree to use it anywhere than it should be changed in  \build\media_source\system\scss\fields\switcher.scss maybe make a seperate PR for this change

**Comment by brianteeman** [comment]
https://github.com/joomla/joomla-cms/pull/27926#issuecomment-586359559

**Comment by korenevskiy** [comment]
@angieradtke see this my sketch sketches.
Joomla 4 Editor 2
Joomla 4 Editor
Joomla 4 Articles

**Comment by korenevskiy** [comment]
I also suggest making switches (Yes/No)
GIF Вс 16 02 2020 4-12-07

I absolutely agree that the top status bar should be blue.
My drafts are slightly different. Since all the styles I did in the Internet browser panel. After changing the status, the page reloads to the default styles and my modifications disappear. 
I don't know how to use SASS, I can offer my own styles in CSS.

**Comment by bembelimen** [comment]
> 
> 
> Do we need the sidebar on the login-screen?

> 
> 
> Personally not and the links are silly. But others were very insistent on that

For example me :) The sidebar is not about the links, it's about the possibility to add content there. Giving advices for your customers etc. The links are just placeholders so to speak (which could help anyways)

**Comment by bembelimen** [comment]
> 
> 
> I also suggest making switches (Yes/No)
> GIF Вс 16 02 2020 4-12-07

The problem with included texts (which was a benefit from, I guess Brian did it, the current solution) is, that you're very limited in text length.

**Comment by Bakual** [comment]
> The problem with included texts (which was a benefit from, I guess Brian did it, the current solution) is, that you're very limited in text length.

The button itself should not have long texts anyway. The "what" should be in the label, the button should be simple "yes/no".

**Comment by brianteeman** [comment]
Included texts is not going to be realistically possible in a multilanguage system. It places too great a restriction on the use.

**Comment by richard67** [comment]
In the [example above](https://github.com/joomla/joomla-cms/pull/27918#issuecomment-586685968) I would make a group with label "Notify before" and then for the particular buttons use lables "day", "5 days" and so on, and use "yes" and "no" for the values.

The values should be so that not only the color gives the information if on or off but also some change of text. The position of the handle left or right is in my opinion also not enough info in addition to the color. I.e. if in the example above you are color blind and don't know about the meaning of the position of the handle, you don't know if it is on or off.

**Comment by brianteeman** [comment]
"Notify before" would not be a label - it would be part of a sentence. That is not good

Your comments about the use of color to convey meaning are also correct. We should not.

**Comment by richard67** [comment]
@brianteeman 
```
Notify before ...
... day:
... 5 days:
```
would complete the sentence.

But I agree, it's not nice.

**Comment by brianteeman** [comment]
which does not work if you are using a screen reader - therefore it is wrong ;)

**Comment by Bakual** [comment]
Before making changes to the switcher to make it look like button groups, we could as well just use button groups: https://github.com/joomla/joomla-cms/pull/27952 😄

**Comment by brianteeman** [comment]
https://github.com/joomla/joomla-cms/pull/27918#issuecomment-586360460

**Comment by angieradtke** [comment]
> 
> 
> @angieradtke see this my sketch sketches.
> Joomla 4 Editor 2
> Joomla 4 Editor
> Joomla 4 Articles

the border-radius ist located in a variable and so easy to change  .-)

**Comment by richard67** [comment]
@angieradtke I've just made PR [https://github.com/angieradtke/joomla-cms/pull/1](https://github.com/angieradtke/joomla-cms/pull/1) to your repo to update this branch to latest 4.0-dev including solve merge conflicts. Check the description and my comment there about how to proceed. If all ok, merge, else ping me here or in that PR there or by email.

**Comment by richard67** [comment]
@brianteeman I have helped @angieradtke with solving merge conflicts to 4.0-dev so this PR is up to date now. One of these conflicts was between changes for `a[role=tab]` and those your recently merged PR #27860 in file `administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss`. Could you check if I have solved this conflict in the right way, i.e. if this PR here is right now or if your change from #27860 has to be applied again, and if necessary make a PR to Angie's branch of this PR? Thanks in advance for helping.

**Comment by richard67** [comment]
@infograf768 I have helped @angieradtke with solving merge conflicts to 4.0-dev so this PR is up to date now. One of these conflicts was between changes for `.custom-select` and those your recently merged PR #27864 in file `administrator/templates/atum/scss/vendor/bootstrap/_custom-forms.scss`. Could you check if I have solved this conflict in the right way, i.e. if this PR here is right now or if your change from #27864 has to be applied again, and if necessary make a PR to Angie's branch of this PR? Thanks in advance for helping.

**Comment by richard67** [comment]
@angieradtke The font size of the Update Check quick-icons on the admin panel is too big for the button and also too big compared with the other icons, e.g. System icons at the top:

angieradtke-admintemplate-quickicons-font-size

Maybe some selector needs to be modified in file `administrator/templates/atum/scss/blocks/_quickicons.scss` to get the sasme, smaller `.875rem` font size (I think its variable value `$quickicon-icon-size`), but I haven't already found the right one.

Do you think you can fix that?

**Comment by brianteeman** [comment]
@richard67 This is a proof of concept - work in progress and as stated
> Most changes made in the edit-article-view and at the sidenav.

So please help @angieradtke out by not commenting on anything other than that she asks for feedback on

**Comment by angieradtke** [comment]
Thanks a lot guys  .-)
   At the moment 'm fighting with my local installation. It is broken 
after I  took the pull request.

Error: Class 'Joomla\Module\Menu\Site\Helper\MenuHelper' not found: 
Class 'Joomla\Component\Actionlogs\Administrator\Plugin\ActionLogPlugin' 
not found

I'm not sure what  to do best  solve this issue issue.


Am 16.02.2020 um 17:23 schrieb Brian Teeman:

**Comment by brianteeman** [comment]
go to the libraries folder  and delete autoload_psr4.php

Then refresh and it will be rebuilt

**Comment by richard67** [comment]
> go to the libraries folder and delete autoload_psr4.php
> 
> Then refresh and it will be rebuilt

Confirmed. Brian was just 10 seconds faster than me.

**Comment by angieradtke** [comment]
Hallelujah .-)  thank you  Brian .-)

Am 16.02.2020 um 17:39 schrieb Brian Teeman:

**Comment by angieradtke** [comment]
edit

is your article-edit Layout broken as well?

not only that
edit2

**Comment by SharkyKZ** [comment]
Run `npm i`.

**Comment by richard67** [comment]
@angieradtke If Sharky's hint in previous comment is not enough let me know, then I can provide more steps to clean up.

**Comment by angieradtke** [comment]
supi, thank you  it works .-)

Time for CSS .-)

Am 16.02.2020 um 18:04 schrieb SharkyKZ:

**Comment by infograf768** [comment]
@angieradtke 
please correct atum template-rtl.scss

It should be
```
    [class*=' fa-'],
    [class^='fa-'],
    [class^='icon-'],
    [class*=' icon-'],
    [class^='#{$fa-css-prefix}-'],
    [class*=' #{$fa-css-prefix}-'] {
      margin: 0 -1.625rem 0 1rem;
    }
```
instead of
```
    [class*=' fa-'],
    [class^='fa-'],
    [class^='icon-'],
    [class*=' icon-'],
    [class^='#{$fa-css-prefix}-'],
    [class*=' #{$fa-css-prefix}-'] {
      margin: 0 -27px 0 27px;
    }
```
This will correct the Help and Options buttons (as well as other similar type of buttons)
from

Screen Shot 2020-02-17 at 07 29 03


to

Screen Shot 2020-02-17 at 07 57 10

Also, please modify _header.scss to 
```
  .badge {
    position: absolute;
    top: 0.4rem;

    [dir=ltr] & {
      left: 50%;
      margin-left: 0.8rem;
    }

    [dir=rtl] & {
      right: 50%;
      margin-right: 0.8rem;
    }
  }

  .fa-angle-down {
    position: absolute;
    top: 0.9rem;

    [dir=ltr] & {
      left: 50%;
      margin-left: -1.8rem;
    }

    [dir=rtl] & {
      right: 50%;
      margin-right: -1.8rem;
    }
  }
```

in order to separate more the angle-down icon from the related icon and correct RTL

Before change

Screen Shot 2020-02-17 at 08 24 34

after change we will get for LTR

Screen Shot 2020-02-17 at 08 18 04

for RTL

Screen Shot 2020-02-17 at 08 17 40

**Comment by angieradtke** [comment]
Hi JM,

still the master of RTL .-)

  done - thank you .-)

Am 17.02.2020 um 08:29 schrieb infograf768:

**Comment by infograf768** [comment]
👍

**Comment by angieradtke** [comment]
adding gradients to the quickicons 
gradients

**Comment by angieradtke** [comment]
changed side-nav
sidenav

**Comment by angieradtke** [comment]
error

**Comment by brianteeman** [comment]
Probably not - what error are you getting

File: administrator/templates/atum/scss/blocks/_quickicons.scss [hunk 2]
```diff
@@ -155,6 +178,7 @@
       text-align: center;
       min-height: 1.2rem;
       padding: 0.2rem;
+      font-size:.875rem;
```
Comment (infograf768) [inline administrator/templates/atum/scss/blocks/_quickicons.scss]: space after :

File: administrator/templates/atum/scss/blocks/_sidebar-nav.scss [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 .sidebar-nav {
 
+  /* override bootstrap important  */
+  background:var(--atum-sidebar-bg) !important;
```
Comment (infograf768) [inline administrator/templates/atum/scss/blocks/_sidebar-nav.scss:6]: space after :

File: administrator/templates/atum/scss/blocks/_sidebar-nav.scss [hunk 4]
```diff
@@ -10,21 +14,50 @@
     margin-bottom: .5rem;
     font-size: .9rem;
     font-weight: $font-weight-bold;
+    border-bottom:solid 1px var(--atum-bg-light);
```
Comment (infograf768) [inline administrator/templates/atum/scss/blocks/_sidebar-nav.scss:17]: space after :

File: administrator/templates/atum/scss/blocks/_sidebar-nav.scss [hunk 5]
```diff
@@ -10,21 +14,50 @@
...
     &.divider {
       margin: .3rem 0;
+      border-bottom: 0;
+
     }
 
     a {
-      color: var(--atum-link-color);
+      color: var(--atum-text-dark);
+      display:block;
```
Comment (infograf768) [inline administrator/templates/atum/scss/blocks/_sidebar-nav.scss:36]: space after:

File: administrator/templates/atum/scss/blocks/_sidebar-nav.scss [hunk 6]
```diff
@@ -10,21 +14,50 @@
...
+      padding:  0.35rem 0.35rem 0.35rem 1.2rem;
```
Comment (infograf768) [inline administrator/templates/atum/scss/blocks/_sidebar-nav.scss:37]: one space too much after :

**Comment by brianteeman** [comment]
@infograf768 please let @angieradtke work on the template and leave the comments about code style until she is finished and then they can all be fixed in one go. Many automatically by the lint

File: administrator/templates/atum/scss/blocks/_sidebar-nav.scss [hunk 7]
```diff
@@ -10,21 +14,50 @@
...
+      position: relative;
+      overflow: hidden;
+      font-weight:normal;
+
+      &:before
+      {
+        position:absolute;
+        left:0;
+        top:0.75rem;
```
Comment (infograf768) [inline administrator/templates/atum/scss/blocks/_sidebar-nav.scss:46]: line 40, 44, 45, 46 space after :

**Comment by brianteeman** [comment]
Could you take a look at all the instances of transitions like `.2s ease-in-out` on the icons please

I find it very distracting that icons change on hover at a different time to the associated text. Either there should be no transition or the transition should be applied to the text as well as the icon

## current 
with

## without any transition
none

**Comment by brianteeman** [comment]
That's great - thanks - can you look at the toolbar buttons as well please

toolbar

**Comment by N6REJ** [comment]
@angieradtke to help make it clearer for people it's a draft you might change your title to 
**[4.0] [DRAFT] UX improvements admintemplate**

**Comment by brianteeman** [comment]
thx @angieradtke

**Comment by angieradtke** [comment]
I spent a week working intensively with the autum-template.
At the beginning I  had to understand the basic structure and setup. 
 I'm not very familiar with Git and I certainly made a lot of code style mistakes.
But I still hope that I could help.

I think the basis of the template is quite good, otherwise I wouldn't have been able to make so many adjustments so quickly. 

I would like to finish this pull request by now, because otherwise it gets too big and I don't know if I marched in the right direction at all.

This is primarily a proposal for the visual design
There are still many views to be edited and adjusted. 

It is certainly important for many uninvolved people to know that the color  is controlled by variables and  therefore customisable. The color of the header can be changed as well as the color of all links. etc. This should be communicated to steer the discussions about colors in the right direction.

It is important now to  decide together on a basic colour concept that creates the broadest possible consensus.  I think blue is always good.
However, when choosing the colour shade, we have to keep the colour contrasts ( Accessibilty) in mind. I have now made a proposal here, which is to be checked for approval.
 
When designing the input fields, I am not sure about the choice of borders and boxshadows. We need a good balance between lightness and enough contrast.  
It's not that simple. To keep us flexible here, we find both values as variables. 

I couldn't take a look at al views yet and have worked on the top level first. What bothers me now is the unsightly display of the error messages. But this can be changed first when the HTML output is unified.  Sometimes the code comes from the php-File sometimes from the JS.
I would like the surrounding <html> elements to be unified and only output when they are needed.  Afterwards I think it would be very important to revise the media manager. 

Basically all files should be checked for missing or unnecessary variables. This can become quite complex. I hope I could help for now and thank Brian for his input.

File: layouts/joomla/searchtools/default/bar.php [hunk 8]
```diff
@@ -25,13 +24,25 @@
...
 $filterButton = $data['options']->get('filterButton', true);
 $searchButton = $data['options']->get('searchButton', true);
-
 $filters = $data['view']->filterForm->getGroup('filter');
 ?>
 
 <?php if (!empty($filters['filter_search'])) : ?>
 	<?php if ($searchButton) : ?>
-		<div class="btn-group mr-2">
+	    <div>
```
Comment (richard67) [inline layouts/joomla/searchtools/default/bar.php]:
```suggestion
		<div>
```

**Review by richard67 — COMMENTED** [review]
Tabs, not spaces

**Comment by rdeutz** [comment]
I want to merge this PR as fast as possible, I know there are issues and we might break things on other plances, but I do think that we can work better parallel when we merge the state we have now. So if someone sees big show stopper, we have to fix before I merge this, please let me know.  

In the meantime I will have a look why the test are failing.

**Comment by brianteeman** [comment]
>  if someone sees big show stopper, we have to fix before I merge this, please let me know.

I am still not in favour of changing the switcher back to two buttons. I would be much happier if that was removed from here and dealt with as a separate issue.

**Comment by ciar4n** [comment]
> I want to merge this PR as fast as possible, I know there are issues and we might break things on other plances, but I do think that we can work better parallel when we merge the state we have now. So if someone sees big show stopper, we have to fix before I merge this, please let me know.

The same was done with the last major update of this template. Atum CSS is full of redundant rules, with every iteration just adding more CSS with little consideration for the CSS codebase as a whole. I'm not saying this PR is the same but I know in the last major PR, a fraction of these made redundant rules were pointed out. The PR was merged regardless, using the same excuse, them same redundant rules are still there.

**Comment by angieradtke** [comment]
> them same redundant rules are still there.

 I wrote that as well and everybody would be thankful if you can clean it up with your expertise

File: administrator/components/com_content/tmpl/article/edit.php [hunk 9]
```diff
@@ -61,7 +61,7 @@
 		<?php echo HTMLHelper::_('uitab.addTab', 'myTab', 'general', Text::_('COM_CONTENT_ARTICLE_CONTENT')); ?>
 		<div class="row">
 			<div class="col-lg-9">
-				<div class="card">
+				<div class="">
```
Comment (brianteeman) [inline administrator/components/com_content/tmpl/article/edit.php]: looks like an error to me

File: layouts/joomla/searchtools/default/bar.php [hunk 10]
```diff
@@ -25,13 +24,25 @@
...
 $filterButton = $data['options']->get('filterButton', true);
 $searchButton = $data['options']->get('searchButton', true);
-
 $filters = $data['view']->filterForm->getGroup('filter');
 ?>
 
 <?php if (!empty($filters['filter_search'])) : ?>
 	<?php if ($searchButton) : ?>
-		<div class="btn-group mr-2">
+		<div>
```
Comment (brianteeman) [inline layouts/joomla/searchtools/default/bar.php]: looks like this div is not required

**Comment by brianteeman** [comment]
As this template PR is 99% scss and not rewriting the php (again) there really isn't such a big rush to merge it. Anyone can take @angieradtke branch and submit pull requests there so that it is really great when it is merged here. (I know that's what I have been doing)

**Comment by angieradtke** [comment]
this <div> grouped the elements inside, because  of  the flexbox -usage  
of the wrapping element .

Am 19.02.2020 um 12:11 schrieb Brian Teeman:

**Comment by infograf768** [comment]
RTL bug:
The `select-bg.svg` has been modified, but not the `select-bg-rtl.svg` one.

Therefore we get for LTR

Screen Shot 2020-02-19 at 12 15 35

and for RTL

Screen Shot 2020-02-19 at 12 15 04

**Comment by infograf768** [comment]
Here is a corrected rtl file

[select-bg-rtl.svg.zip](https://github.com/joomla/joomla-cms/files/4224815/select-bg-rtl.svg.zip)

**Comment by infograf768** [comment]
we could add to the rtl.svg the code `preserveAspectRatio="xMinYMin meet"` which I forgot in the file above.

**Comment by angieradtke** [comment]
@infograf768 @brianteeman now I did something wrong merged what I don't want merge????

File: administrator/components/com_content/tmpl/article/edit.php (hunk 9 above)
Comment (angieradtke) [reply administrator/components/com_content/tmpl/article/edit.php]: I removed the class because I don't want the box-shadow there

Comment (brianteeman) [reply administrator/components/com_content/tmpl/article/edit.php]: then you should just have a plain `<div>` and not the empty `class=""`

**Comment by rdeutz** [comment]
> As this template PR is 99% scss and not rewriting the php (again) there really isn't such a big rush to merge it. Anyone can take @angieradtke branch and submit pull requests there so that it is really great when it is merged here. (I know that's what I have been doing)

I want to allow to work in parallel and not let anyone work on this branch, smaller changes are better to discuss and I belive we would processing faster.

**Comment by brianteeman** [comment]
All of that can be done on a different branch or on angie's fork

**Comment by Quy** [comment]
Not aligned.

filter

**Comment by brianteeman** [comment]
its the padding on the selects being different when it is using choices.js

**Comment by angieradtke** [comment]
Hi Guys,

I don't know what do do now, waiting  .....

**Comment by TobsBobs** [comment]
Should we test this PR or will it merged direclty?

**Comment by richard67** [comment]
@angieradtke You mean you have finished works? If so, just remove the "[WIP]" from the title of this PR and post back here that it is ready for test, and then we test it.

Or is there still something to do and you wait for advise?

**Comment by richard67** [comment]
Ah and @rdeutz wanted to have a look why Drone tests are failing.

**Comment by rdeutz** [comment]
We talked about this PR in a meeeting about the Beta blockers and we decided to pull some things out of this PR, fix testing and merge it then. Plan is to do this over the weekend.

**Comment by richard67** [comment]
@rdeutz This weekend or next weekend?

**Comment by rdeutz** [comment]
this, so today or at least tomorrow

**Comment by HLeithner** [comment]
> this, so today or at least tomorrow

Roberts weekend starts on Friday and ends on Monday ;-)

File: administrator/components/com_content/tmpl/articles/default.php [hunk 11]
```diff
@@ -93,7 +93,7 @@
 				?>
 				<?php if (empty($this->items)) : ?>
 					<div class="alert alert-info">
-						<span class="fas fa-info-circle" aria-hidden="true"></span><span class="sr-only"><?php echo Text::_('INFO'); ?></span>
+						<span class="fa fa-info-circle" aria-hidden="true"></span><span class="sr-only"><?php echo Text::_('INFO'); ?></span>
```
Comment (richard67) [inline administrator/components/com_content/tmpl/articles/default.php]:
```suggestion
						<span class="fas fa-info-circle" aria-hidden="true"></span><span class="sr-only"><?php echo Text::_('INFO'); ?></span>
```

File: administrator/components/com_content/tmpl/articles/default.php [hunk 12]
```diff
@@ -216,7 +216,7 @@
 									}
 									?>
 									<span class="sortable-handler<?php echo $iconClass ?>">
-										<span class="fas fa-ellipsis-v" aria-hidden="true"></span>
+										<span class="fa fa-ellipsis-v" aria-hidden="true"></span>
```
Comment (richard67) [inline administrator/components/com_content/tmpl/articles/default.php]:
```suggestion
										<span class="fas fa-ellipsis-v" aria-hidden="true"></span>
```

**Review by richard67 — CHANGES_REQUESTED** [review]
@rdeutz The changes of `fas` to `fa` class prefix for fontawesome should be reverted, see PR #27657 .

**Review by richard67 — APPROVED** [review]

**Comment by brianteeman** [comment]
woohooo

**Comment by Hackwar** [comment]
👍

**Comment by ciar4n** [comment]
No tests required anymore?

**Comment by N6REJ** [comment]
> @rdeutz The changes of `fas` to `fa` class prefix for fontawesome should be reverted, see PR #27657 .

oh gawd PLEASE don't undo what I've started

File: administrator/templates/atum/scss/blocks/_sidebar.scss [hunk 13]
```diff
@@ -167,7 +169,7 @@
       .menu-quicktask {
         > a {
           &.mm-active {
-            background-color: var(--atum-bg-dark-80);
+          /*  background-color: var(--atum-bg-dark-80); */
```
Comment (HLeithner) [inline administrator/templates/atum/scss/blocks/_sidebar.scss:172]: Please remove code that is no longer used.

**Comment by HLeithner** [comment]
> > @rdeutz The changes of `fas` to `fa` class prefix for fontawesome should be reverted, see PR #27657 .
> 
> oh gawd PLEASE don't undo what I've started

Could you point me to the change? didn't found it at a quick look

File: administrator/templates/atum/scss/blocks/_sidebar.scss (hunk 13 above)
Comment (astridx) [reply administrator/templates/atum/scss/blocks/_sidebar.scss:172]: Please see https://github.com/joomla/joomla-cms/pull/28035

Comment (HLeithner) [reply administrator/templates/atum/scss/blocks/_sidebar.scss:172]: thanks

**Comment by richard67** [comment]
> > > @rdeutz The changes of `fas` to `fa` class prefix for fontawesome should be reverted, see PR #27657 .
> > 
> > 
> > oh gawd PLEASE don't undo what I've started
> 
> Could you point me to the change? didn't found it at a quick look

@HLeithner Is resolved since a while.

**Comment by HLeithner** [comment]
Then I'm late to the party^^ thx richard

**Comment by C-Lodder** [comment]
Why does the core backend template have template overrides? Surely the is the road towards a maintainers nightmare?

**Comment by brianteeman** [comment]
it is - and it really shouldnt - the searchtools one has already been pulled

**Comment by brianteeman** [comment]
@C-Lodder see #28052
